testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -29,6 +29,12 @@
     SUPPORTS_TOPN_PUSHDOWN_WITH_VARCHAR(fallback -> fallback.test(SUPPORTS_TOPN_PUSHDOWN) && fallback.test(SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY)),
 
     SUPPORTS_AGGREGATION_PUSHDOWN,
+    // Most connectors don't support aggregation pushdown for statistical functions
+    SUPPORTS_AGGREGATION_PUSHDOWN_STDDEV(false),
+    SUPPORTS_AGGREGATION_PUSHDOWN_VARIANCE(false),
+    SUPPORTS_AGGREGATION_PUSHDOWN_COVARIANCE(false),
+    SUPPORTS_AGGREGATION_PUSHDOWN_CORRELATION(false),
+    SUPPORTS_AGGREGATION_PUSHDOWN_REGRESSION(false),
```

Comment:
Most connectors don't support aggregation pushdown either. 
Did you consider defaulting these to `SUPPORTS_AGGREGATION_PUSHDOWN`?

Reply:
The connectors which support aggregation pushdown mostly support until `STDDEV` and `VARIANCE`. The lower 3 are not supported almost everywhere - helps keep the `hasBehaviour` small.

I can change it to make the `hasBehaviour` more explicit about what's NOT supported. I don't have a strong opinion on this.

Reply:
fine as is, can revisit if needed